src/Features/CSharp/Portable/Wrapping/InitializerExpression/CSharpInitializerExpression.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.Wrapping.InitializerExpression
+{
+    internal partial class CSharpInitializerExpression : AbstractCSharpInitializerExpressionWrapper<InitializerExpressionSyntax, ExpressionSyntax>
+    {
+        protected override SeparatedSyntaxList<ExpressionSyntax> GetListItems(InitializerExpressionSyntax listSyntax)
+        {
+            return listSyntax.Expressions;
+        }
+
+        protected override bool PositionIsApplicable(SyntaxNode root, int position, SyntaxNode declaration, InitializerExpressionSyntax listSyntax)
+        {
+            var startToken = listSyntax.GetFirstToken();
+
+            var token = root.FindToken(position);
+            if (token.Parent.Ancestors().Contains(listSyntax))
+            {
+                var current = token.Parent;
+                while (current != listSyntax)
+                {
+                    if (CSharpSyntaxFactsService.Instance.IsAnonymousFunction(current))
+                    {
+                        return false;
+                    }
+
+                    current = current.Parent;
```

Comment:
why is all this code needed?

Reply:
I misunderstood what was happening here. I thought it was required to figure out if we should offer the refactoring when inside of the initializer. From my testing, that wasn't the case, so I removed it.